### PR TITLE
Dockerfile: Bump ostreeuploader

### PR DESCRIPTION
**Pull request by doanac**
Brings in functionality required by MEDs. The newest version requires go.19 to compile

### CODE REVIEW

**Comment by doanac** [comment]
I've tested this container [doanac/lmp-sdk ](https://hub.docker.com/r/doanac/lmp-sdk) against my factory and all is working:

https://ci.foundries.io/projects/andy-corp/lmp/builds/9/raspberrypi4-64/artifacts/console.log

**Comment by ricardosalveti** [comment]
Cool, is it safe for all factories?

File: Dockerfile [hunk 1]
```diff
@@ -18,12 +18,12 @@ RUN git clone https://github.com/containers/skopeo.git /skopeo && \
 FROM ubuntu:20.04 AS fiotools
 RUN apt-get update
 RUN apt-get install -y wget git gcc make -y
-RUN wget -P /tmp https://go.dev/dl/go1.18.linux-amd64.tar.gz && \
-    tar -C /usr/local -xzf /tmp/go1.18.linux-amd64.tar.gz
+RUN wget -P /tmp https://go.dev/dl/go1.19.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /tmp/go1.19.linux-amd64.tar.gz
```
Comment (ricardosalveti) [inline Dockerfile:22]: Better to use 1.19.8, no? (latest stable release)

Comment (doanac) [reply Dockerfile:22]: we can wait till Tuesday for 1.19.9: 
```
go1.19.9 (planned for 2023-05-02) will include security fixes to the standard library, as well as bug fixes to the compiler, the runtime, and the crypto/tls and syscall packages. 
```

**Comment by doanac** [comment]
It is safe for all factories.

File: Dockerfile (hunk 1 above)
Comment (ricardosalveti) [reply Dockerfile:22]: Cool, better wait for 1.19.9 then.

Comment (ricardosalveti) [reply Dockerfile:22]: @doanac mind updating the pr with .9?

Comment (doanac) [reply Dockerfile]: force pushed the 1.19.9 version of golang

**Review by quaresmajose — APPROVED** [review]

**Comment by ricardosalveti** [comment]
Merged manually to avoid another run (as I had other changes to push).

**Pull request closed without merging**